Resources/views/CRUD/list_array.html.twig

```diff
@@ -8,19 +8,10 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% import _self as list %}
-{%  macro render_array(value) %}
-    {% for key, val in value %}
-        {% if val is iterable %}
-            [{{ key }} => {{ list.render_array(val) }}}]
-        {%  else %}
-            [{{ key }} => {{ val }}]
-        {%  endif %}
-    {% endfor %}
-{% endmacro %}
+{% import 'SonataAdminBundle:CRUD:base_array_macro.html.twig' as list %}
 
 {% extends admin.getTemplate('base_list_field') %}
 
 {% block field %}
-    {{ list.render_array(value) }}
+    {{ list.render_array(value, field_description.options.inline is not defined or true == field_description.options.inline) }}
```

Comment:
If doing this, when you set `'inline' => false`  in your field options, it will finally be set to `true` because `false` is considered as empty in twig `default` method, no ?